### ISSUE TITLE
Update and redraw the tree

### ISSUE THREAD

**Issue by amehlen** [amsik/liquor-tree#36]
Hello,

I have a dropdown called project on the parent component and a child component to display the tree.  
The treeData is depending on the selected project. The data is fetched from a backend and the data is passed as props from parent to child. Actually a watcher is implemented to verify the data.

How is it possible to update and redraw the tree?

Best regards,
amehlen

**Comment by amsik** [comment]
Hello!

You can use https://amsik.github.io/liquor-tree/#Integration-with-Vuex

**Comment by amehlen** [comment]
Thanks for the lead. I will give it a try later. Actually I found another working solution.
Source: [https://stackoverflow.com/questions/47855365/vue-element-ui-tree-emit-reload-event](url)

Other questions:
- With the help of  issue #5 I get the name of a selected node. 
How is it possible to load the data from the selected node in a form? Is there an event like `@change` or `@click` from Vue?
- If I select a node, is it possible not to expand the children? I want to expand the children only when I clicked on the arrow

Best regards,
amehlen
